### DISCUSSION TITLE
Military outpost armory nerf

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6352] [Ideas and suggestions]
Not many people know how to open these, but if you manage to open the secure cabinets in the military outposts (which can be done pretty easily with certain techniques)  you will be received by the locker with enough handheld weapons and ammo for basically the whole campaign. Every loot table in the game seems kinda ridiculous currently but this one seems to be the one that ruins most of the "scavanger" experience, and a small amount of people give feedback about it.

So i suggest changing the loot table to make it similar to the armories in pirate ships and abandoned outposts.

**Comment by Cogs16** [comment]
Alternatively, it might be fair to keep the loot table as is and make it harder to get to. I.e have a guard whos stationary and dedicated to protecting the armory. Additionally, (and I know this is sort of a low priority hopes and dreams idea) it could be neat to have a guard that monitors CCTV cameras all around the military outpost.

**Reply by ghost** [reply]
I agree with this, military outpost security should definitely be more strict.

**Reply by LePiromano** [reply]
the thing is security guards don't get triggered by stun grenades or flash powder so you can handcuff them and they instantly stop being a threat, maybe if they added what cogs suggested and made security npcs impossible to handcuff this wouldn't be a problem, but devs could just nerf the loot pool as i suggested and it'd be efficient too.
